firebase.ios.js

```diff
@@ -171,6 +185,37 @@ firebase.addAppDelegateMethods = function (appDelegate) {
     };
   }
 
+  if (typeof(FIRDynamicLink) !== "undefined") {
+    appDelegate.prototype.applicationContinueUserActivityRestorationHandler = function (application, userActivity, restorationHandler) {
+      var result = false;
+      
+      if (typeof(FIRDynamicLink) !== "undefined" ) {
```

Comment:
This seems a duplicate of line 188. If you want to change it go ahead, otherwise I'll do it after merging this awesome PR 👍

Reply:
Let me double check if I messed up anything while rebasing.

Reply:
No, it's looking good. It's the same if twice, but I did it to follow the pattern that's used with other delegates. Right now `applicationContinueUserActivityRestorationHandler` is only used for one thing, but if something else will need to be added in the future, it will be more trivial and clear.

Reply:
Smart thinking, thanks for the explanation!